src/EA.Weee.Web/Views/Shared/_ViewEvidenceNotePartial.cshtml

```diff
@@ -230,53 +233,59 @@ else
         {
             var category = Model.CategoryValues[i];
 
-        <tr class="govuk-table__row" id="category-row-@category.CategoryId">
-            <td class="govuk-table__cell govuk-!-text-align-left govuk-!-padding-right-1">
```

Comment:
@GuidoWorkDev I can't really see all these differences within GIT but something doesn't look quite right - No changes have been made to the view other than the addition of the moving of transfer history?